apps/estuary/config/integration.exs

```diff
@@ -1,5 +1,13 @@
 use Mix.Config
 
+host =
```

Comment:
do we need to break out this host here?  What is the usecase for taking the host from the environment in the integration config?

Reply:
You're right we probably don't need this. I was copy/pasting.